src/gpu/vulkan/SDL_gpu_vulkan.c

```diff
@@ -10846,6 +10852,17 @@ static Uint8 VULKAN_INTERNAL_CreateInstance(
             VK_EXT_DEBUG_UTILS_EXTENSION_NAME);
     }
 
+    if (renderer->supportsColorspace) {
+        /* Append colorspace extension */
+        instanceExtensionNames[instanceExtensionCount++] =
+            VK_EXT_SWAPCHAIN_COLOR_SPACE_EXTENSION_NAME;
+    } else {
+        SDL_LogWarn(
```

Comment:
I don't think this warning is necessary. It makes sense for the debugUtils extension since you're actively opting into debug mode and should receive some notice if it doesn't work. But for colorspaces, the client can already check for support via the swapchain composition query (which I assume return SDL_FALSE if the colorspace extension is missing?).